utilities.py

```diff
@@ -1,6 +1,11 @@
-# Import statements
+import tensorflow as tf
 
 
 def process_img(input_image):
-    """Placeholder function that will be eventually be used to process our uploaded images into a model-usable form."""
+    """
```

Comment:
I see you've used two different styles of commenting your code. I would suggest we standarize as a team.